src/app/_components/StatusBar/__tests__/StatusBarBase.test.tsx

```diff
@@ -2,7 +2,6 @@ import { system } from '@/ui/theme/theme';
 import { ChakraProvider } from '@chakra-ui/react';
```

Comment:
Consider adding a test here to validate testnet status bar won't show up on mainnet.

Reply:
done. added the test in the banner directory